app/components/Feed/context.js

```diff
@@ -44,3 +53,16 @@ export const contextRender = {
   'notifications.announcement': renderAnnouncement,
   'gallery.gallerypicture': renderGalleryPicture
 };
+
+export function toLink(linkAndText: linkAndText) {
+  return linkAndText.notLink ? (
+    toSpan(linkAndText)
+  ) : (
+    <Link to={linkAndText.link}>{linkAndText.text}</Link>
+  );
+}
+
+export function toSpan(linkAndText: linkAndText) {
```

Comment:
What do you mean with the fact that the parameter `linkToText` has the type `linkToText` (the same name)? Since the global `linkAndText` is an object, i guess that is the reason why flow doesn't care.